_states/ext_state_qvm.py

```diff
@@ -76,11 +100,11 @@ def __virtual__():
     '''
     # XXX: TEMP
     if not hasattr(qubes_utils, '__opts__'):
-        qubes_utils.__opts__ = __opts__
+        qubes_utils.__opts__ = __opts__  # NOQA
```

Comment:
Also take a look how I've dealt with it here: https://github.com/QubesOS/qubes-mgmt-salt-dom0-update/blob/master/_modules/qubes_dom0_update.py#L38-L46